src/Components/Publishing/Display/Canvas/CanvasText.tsx

```diff
@@ -16,13 +17,18 @@ interface DivProps extends React.HTMLProps<HTMLDivElement> {
 
 export const CanvasText: React.SFC<CanvasTextProps> = props => {
   const { disclaimer, unit } = props
+  let resized_logo = resize(unit.logo, {
```

Comment:
We've had a few issues in staging where unfinished display ads are published by the content team, might be worth checking if the unit.logo exists before calling `resize` on it -- not sure what it does when we pass it undefined.